src/ext/NetFxExtension/wixlib/NetFx48.wxs

```diff
@@ -87,13 +87,13 @@
           Compressed="no"
           Name="!(wix.NetFx48RedistPackageDirectory)ndp48-x86-x64-allos-enu.exe">
         <RemotePayload
-          CertificatePublicKey="D17AFCE951979605250FAEBAFA7AE974899AED22" 
-          CertificateThumbprint="C82273A065EC470FB1EBDE846A91E6FFB29E9C12" 
+          CertificatePublicKey="F49F9B33E25E33CCA0BFB15A62B7C29FFAB3880B" 
+          CertificateThumbprint="ABDCA79AF9DD48A0EA702AD45260B3C03093FB4B" 
           Description="Microsoft .NET Framework 4.8 Setup" 
-          Hash="8DD233698D5EB4609B86C2AC917279FE39E0EF4C" 
```

Comment:
It bugs me that attributes that are ignored have to be specified. But I can live with it, given it's been that way for 10 years or so...